blocks/image-compare/editor.scss

```diff
@@ -0,0 +1,30 @@
+
+/* main block wrapper */
+.controls {
+	display: flex;
+}
+
+.img-edit-before,
+.img-edit-after {
+	min-width: 180px;
+	width: 100%;
+	max-width: 50%;
+	display: block;
+	text-align: center;
+}
+
+.img-edit-before {
+	border-right: 1px solid #999999;
+}
+
+.caption {
```

Comment:
This is probably now dead CSS. 

We should get into the habit of a nice prefix on a per-block basis. I.e. for Layout Grid it would be `.layout-grid__caption-style`. In this case, it would be `.image-compare__caption-style`. Outside of being more readable code, it ensures we don't accidentally use a CSS class that a ton of other random things like WordPress themes use, of which there would be a high likelyhood here.